curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/selecting-from-many-options-with-switch-statements.english.md

```diff
@@ -9,19 +9,15 @@ forumTopicId: 18277
 ## Description
 <section id='description'>
 If you have many options to choose from, use a <code>switch</code> statement. A <code>switch</code> statement tests a value and can have many <code>case</code> statements which define various possible values. Statements are executed from the first matched <code>case</code> value until a <code>break</code> is encountered.
-Here is a <dfn>pseudocode</dfn> example:
+A <code>switch</code> statement can be written as follows:
```

Comment:
```suggestion
Here is an example of a <code>switch</code> statement:
```
This does not define the syntax, exactly, so it's probably best to reinforce that it is just an example.